### PR TITLE
Update R runtime files

### PR DESCRIPTION
**Pull request by jalvesaq**
- Fix indentation issue with ggplot().
- Setlocal autoindent in indent/r.vim.
- New syntax option: rmd_include_latex.
- Clear syn iskeyword to recognize _ as keyword.
- Document some options.

### CODE REVIEW

**Comment by jalvesaq** [comment]
The `vimtags` test is failing. I'm not sure if I have to run `:helptags` to update the `tags` file and include it in the pull request.

If I run `:helptags`, I get this diff:

```diff
diff --git a/runtime/doc/tags b/runtime/doc/tags
index 5c127b23b..89078992b 100644
--- a/runtime/doc/tags
+++ b/runtime/doc/tags
@@ -7302,6 +7302,7 @@ ft-rego-syntax	syntax.txt	/*ft-rego-syntax*
 ft-rexx-syntax	syntax.txt	/*ft-rexx-syntax*
 ft-rmd-plugin	filetype.txt	/*ft-rmd-plugin*
 ft-rmd-syntax	syntax.txt	/*ft-rmd-syntax*
+ft-rnoweb-plugin	filetype.txt	/*ft-rnoweb-plugin*
 ft-rrst-plugin	filetype.txt	/*ft-rrst-plugin*
 ft-rrst-syntax	syntax.txt	/*ft-rrst-syntax*
 ft-rst-plugin	filetype.txt	/*ft-rst-plugin*
@@ -7920,7 +7921,6 @@ help-buffer-options	helphelp.txt	/*help-buffer-options*
 help-context	help.txt	/*help-context*
 help-curwin	tips.txt	/*help-curwin*
 help-summary	usr_02.txt	/*help-summary*
-help-tags	tags	1
 help-translated	helphelp.txt	/*help-translated*
 help-writing	helphelp.txt	/*help-writing*
 help-xterm-window	helphelp.txt	/*help-xterm-window*
```

**Comment by chrisbra** [comment]
yes, please run `make tags` in the `runtime/doc` directory and add the changed tags file to the PR.

File: runtime/syntax/rmd.vim [hunk 1]
```diff
@@ -225,6 +259,10 @@ unlet s:save_pandoc_lngs
 let &cpo = s:cpo_save
 unlet s:cpo_save
 
+if has("patch-7.4.1142")
+  syntax iskeyword clear
```
Comment (chrisbra) [inline runtime/syntax/rmd.vim]: >   syntax iskeyword clear

1) I don't think we need this. 
2) we can leave out the test for patch 7.4.1142, by now Vim should always have it.

**Comment by chrisbra** [comment]
can you please also update the last changed header for the doc files? (See the very first line)

**Comment by jalvesaq** [comment]
Thank you for the guidance, @chrisbra!

**Comment by jalvesaq** [comment]
Sorry, actually I made a mistake and will fix it now.

**Comment by jalvesaq** [comment]
The mistake is fixed. I hope everything is OK now. Thanks!

**Comment by chrisbra** [comment]
thanks!

**Pull request merged**